tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from tests.test_fixtures import *
```

Comment:
Предлагаю несколько решений для данного импорта. Во первых, изменить название модуля на `fixtures.py`, а во вторых, не импортировать их в данный модуль, а прикрутить при помощи `pytest_plugins` (Предлагаю решение ниже, путь может быть другим).
```suggestion
pytest_plugins = "tests.fixtures"
```